src/pyscaffoldext/dsproject/templates/readme_md.template

```diff
@@ -18,23 +18,23 @@ In order to set up the necessary environment:
    ```
    conda activate ${name}
    ```
-3. install `${name}` with:
-   ```
-   python setup.py install # or `develop`
-   ```
+
+> **_NOTE:_**  The conda environment will have ${name} installed in editable mode.
+> Some changes, e.g. in `setup.cfg`, might require you to run `pip install -e .` again.
+
```

Comment:
Question: is it worthy to add a note for the user to review `environment.yml` before creating the env?
I mean, if someone is doing simple data analisis/processing with panda and scikit-learn, pulling up pytorch might be a bit too much (specially because it take some time).

Reply:
Just as an example:

```diff
- 1. create an environment `${name}` with the help of [conda],
+ 1. Review your favourite tools in `environment.yml`
+     and create an environment `${name}` with the help of [conda],
```

Reply:
Good point!